apisupport/apisupport.project/src/org/netbeans/modules/apisupport/project/spi/BrandingModel.java

```diff
@@ -719,7 +719,7 @@ private String backslashesToSlashes (String text) {
         for (BundleKey bundleKey : internationalizedResourceBundleKeys) {
             String localizedBundlepath = bundlepath;
             if(!localizedBundlepath.endsWith("_" + this.locale.toString() + ".properties"))
-                localizedBundlepath = bundlepath.replaceAll(".properties", "_" + this.locale.toString() + ".properties");
```

Comment:
dot in ".properties" should have been escaped: "\.properties" (multiple occurrences in this class)